src/hooks/useOpenSavings.js

```diff
@@ -1,18 +1,16 @@
 import { useCallback } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { setOpenSavings } from '../redux/openStateSettings';
+import { useMMKVBoolean } from 'react-native-mmkv';
+import useAccountSettings from './useAccountSettings';
 
 export default function useOpenSavings() {
-  const dispatch = useDispatch();
-
-  const isSavingsOpen = useSelector(
-    ({ openStateSettings: { openSavings } }) => openSavings
+  const { accountAddress: address } = useAccountSettings();
```

Comment:
nit: there seems to be no added benefit of renaming `accountAddress` to `address` and we prefer to use `accountAddress` for clarity when possible